### PR TITLE
initial commit for LP-SRE team ACS access.

### PR DESCRIPTION
**Pull request by Nanyte25**
### What type of PR is this?
_(bug/feature/cleanup/documentation)_

### What this PR does / why we need it?

https://issues.redhat.com/browse/MTSRE-1084

### Which Jira/Github issue(s) this PR fixes?
https://issues.redhat.com/browse/MTSRE-1084
_Fixes #_

### Pre-checks (if applicable):
- [x] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

File: deploy/backplane/lpsre/acs/02-acs-lpsre-admins-project.ClusterRole.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - centrals
+  - securedclusters
+  verbs:
+  - get
+  - list
+  - watch
+
+
+
+
```
Comment (wanghaoran1988) [inline deploy/backplane/lpsre/acs/02-acs-lpsre-admins-project.ClusterRole.yaml:18]: Could you remove those empty lines?

**Comment by Nanyte25** [comment]
@wanghaoran1988 can you rereview thank's

**Comment by Nanyte25** [comment]
@apahim and @jessesarn I am still waiting for this to be merged @jewzaam

**Review by NautiluX — COMMENTED** [review]
/lgtm

**Pull request merged**